java/code/src/com/suse/manager/webui/websocket/Notification.java

```diff
@@ -194,11 +194,12 @@ public static void clearBrokenSessions() {
 
     /**
      * Add a new WebSocket Session to the collection
+     * @param userId authenticated user ID
      * @param session the session to add
      */
-    private static void handshakeSession(User user, Session session) {
+    private static void handshakeSession(Long userId, Session session) {
```

Comment:
```suggestion
    private static void handshakeSession(long userId, Session session) {
```

Reply:
It will put the user ID in a map, so if we receive and long it will to and autoboxing so store the long in the map. I would prefer to keep it receiving the `Long` and avoid that autoboxing. 
Also the code that calls this method already converts it to `Long`.
What do you think @moio ?

Reply:
So here the trade-off is between the delta in performance cost of autoboxing (which might well be zero, if autoboxing is performed just a few lines later, or non-zero, if the caller has the value already boxed and has to un-box it) and the compile-time checking for `null` - a method that accepts `Long` does accept `null`s, while a method that accepts `long` doesn't.

In our huge codebase, I think most occasions to have the compiler check more for us are worth it. There are of course reasonable limits, but I wouldn't worry about performance in this case.